jerry-core/ecma/base/ecma-helpers-value.c

```diff
@@ -199,17 +215,29 @@ ecma_is_value_string (ecma_value_t value) /**< ecma-value */
  *         false - otherwise.
  */
 bool __attr_pure___ __attr_always_inline___
-ecma_is_value_object (ecma_value_t value) /**< ecma-value */
+ecma_is_value_object (ecma_value_t value) /**< ecma value */
 {
   return (ecma_get_value_type_field (value) == ECMA_TYPE_OBJECT);
 } /* ecma_is_value_object */
 
 /**
+ * Check if the value is an error value.
+ *
+ * @return true - if the value contains an error value,
+ *         false - otherwise.
+ */
+bool __attr_pure___ __attr_always_inline___
+ecma_is_value_error (ecma_value_t value) /**< ecma value */
+{
+  return (value & (1 << ECMA_VALUE_ERROR_POS)) != 0;
```

Comment:
I think this should be `return (value & (1ull << ECMA_VALUE_ERROR_POS)) != 0;` like in other places in our code.